### PR TITLE
Feat/local storage

### PR DESCRIPTION
**Pull request by katerinamartysko**

### CODE REVIEW

**Comment by katerinamartysko** [comment]
[Use local storage](https://trello.com/c/l24e6GUq/13-use-local-storage)

File: src/components/Client.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    fetchClientsById(clientID);
+  }, [clientID]);
+
+  if (!client) return null;
+  return (
+    <>
+      <BackButton onRemoveClientID={onRemoveClientID} />
+      <div className={classNames(classes.root, classes.flex)}>
+        <div className={classNames(classes.details, classes.flex)}>
+          <Avatar alt="Remy Sharp" src={client.general.avatar} className={classes.avatar}>
```
Comment (LexaLukovka) [inline src/components/Client.tsx]: Please change to correct the naming for `alt`

File: src/components/ClientsListContainer.tsx [hunk 2]
```diff
@@ -35,16 +51,29 @@ export const ClientsListContainer: FC<Props> = ({ setClientID }) => {
...
     dispatch(setSearch(search));
+  }, 800);
+
+  const handelSearch = (search: string): void => {
+    setClientSearch(search);
+    searchClients(search);
   };
 
+  if (client && isHideClientsList) return null;
   if (isClientsLoading) return <CircularProgress className={classes.load} />;
```
Comment (LexaLukovka) [inline src/components/ClientsListContainer.tsx:63]: I think you need to set `CircularProgress` before checking `client` and `isHideClientsList`

Like this:
`if (isClientsLoading) return <CircularProgress className={classes.load} />;`
`if (client && isHideClientsList) return null;`

**Review by LexaLukovka — CHANGES_REQUESTED** [review]

File: src/components/Client.tsx (hunk 1 above)
Comment (katerinamartysko) [reply src/components/Client.tsx]: Done

File: src/components/ClientsListContainer.tsx (hunk 2 above)
Comment (katerinamartysko) [reply src/components/ClientsListContainer.tsx:63]: Done

File: src/components/Client.tsx [hunk 3]
```diff
@@ -26,15 +26,17 @@ const useStyles = makeStyles()(() => ({
   avatar: {
     width: '160px',
     height: '160px',
+    fontSize: '48px',
     marginRight: theme.spacing(10),
     background: 'pink',
     [theme.breakpoints.down('md')]: {
-      marginBottom: theme.spacing(2),
+      margin: '25px auto',
```
Comment (LexaLukovka) [inline src/components/Client.tsx]: Please use `theme.spacing`

**Review by LexaLukovka — APPROVED** [review]

**Pull request merged**